CMakeLists.txt

```diff
@@ -134,6 +134,10 @@ find_package (Eigen3 3.3 REQUIRED NO_MODULE)
 ### XLNT
 find_package (XLNT REQUIRED)
 
+### Libigl set include path
+include_directories("${CMAKE_PREFIX_PATH}/../build/libigl/include/")
```

Comment:
This won't work if CMAKE_PREFIX_PATH isn't set to the dependencies install directory.  We aren't able to use this for both windows and linux on github actions for example.  We are using CMAKE_PREFIX_PATH=$CONDA_PREFIX on linux.  Something about the alternate gcc version.

On windows we have other issues.

Reply:
Oh I see, so how do we go about setting this include path, or do we have to set the path manually when we install with with libigl in CMakeLists.txt

Reply:
What about having a LibIGL_INCLUDE_PATH which is set by default to "${CMAKE_PREFIX_PATH}/../build/libigl/include/" but can be changes via a flag ?

Reply:
Maybe copy the libigl headers to ${INSTALL_DIR}/include as part of build_dependencies.sh, where all the other dependency headers are stored? Then the include path would always be ${CMAKE_PREFIX_PATH}/include

that seems to be what the [libigl cmake build does by default](https://github.com/libigl/libigl/blob/master/cmake/libigl.cmake#L506), what was the issue with just building libigl via cmake again?

Reply:
There’s nothing to be built. We’re just doing header only mode.

Reply:
I mean the "cmake build process", like what we do with Eigen. Eigen is header-only, but we build it anyway and that just copies the headers to the ${INSTALL_PREFIX}/include directory.

Reply:
Yeah, that would be perfect. Does libgIGL have the same?

Reply:
I tried making the same process for libigl but there are some issues 
for one it it only includes the .h files which cause the codes to complain about the associated .cpp files which are not copied to the install directory.
And of course the build process has it's trouble with managing their own eigen.

I arrieved at this simple solution only after checking all these things :(